app/harvesters/hal/hal_document_type_converter.py

```diff
@@ -1,7 +1,9 @@
-from loguru import logger
+from app.harvesters.abstract_document_type_converter import (
+    AbstractDocumentTypeConverter,
+)
 
 
-class HalDocumentTypeConverter:
+class HalDocumentTypeConverter(AbstractDocumentTypeConverter):
```

Comment:
Here I'd also let the constants outside and set the class attributes values like this :

```
HARVESTER = "HAL"
TYPES_MAPPING = [...]

class HalDocumentTypeConverter(AbstractDocumentTypeConverter):
    
    this.harvester = HARVESTER
    
    this.types_mapping = TYPES_MAPPING